test/types/mongodb.test-d.ts

```diff
@@ -32,3 +32,8 @@ const composedMap = mappedAgg.map<string>(x => x.toString());
 expectType<AggregationCursor<string>>(composedMap);
 expectType<string | null>(await composedMap.next());
 expectType<string[]>(await composedMap.toArray());
+
+// test that VALID_INDEX_OPTIONS like expireAfterSeconds are allowed in IndexDefinitions
+coll.createIndexes([{ key: { event: 1 }, name: 'event', expireAfterSeconds: 2400 }]);
+// test that invalid options are not allowed
+expectError(coll.createIndexes([{ key: { event: 1 }, name: 'event', invalidOption: 2400 }]));
```

Comment:
I added some minimal tests here to ensure the fix worked; would we want to actually test each option in [VALID_INDEX_OPTIONS](https://github.com/mongodb/node-mongodb-native/blob/9e48bbdc95149ca62fa21404624125b87c3c9d56/src/operations/indexes.ts#L28-L58), or does using `expireAfterSeconds` as a representative valid option seem like enough coverage for this scenario?

Reply:
I think we should add tests for them all:
You could use this form: `expectType<IndexDescription>({ key: {}, hidden: true });` not sure if hidden should be allowed here but it is in the VALID set

Reply:
Good call, `version` was also missing as well as `hidden`. PTAL 🙏